docs/usage.md

```diff
@@ -0,0 +1,85 @@
+# Create and send mails
+
+The following is showing an example how you can use HermesBundle to create and send mails.
+
+## Create mail
+
+First of all you need to instantiate a new MailDto object:
+
+```php
+$mail = new MailDto();
+$mail->setTemplate('TEMPLATE_CODE');
+```
+
+To add recipients to the mail, you need to instantiate new ContactDto objects:
+
+```php
+$mail->addTo(new ContactDto($user->getFullName(), $user->getEmail()));
+```
+
+## Add data to the mail
+
+As your template can contains variables, you can pass data to your mail or your recipient.
+Data from both can be used by the templates. If your template defines three variables : `user.firstname`, `user.lastname` and `url`, you must set this three data to your mail (in MailDto or in ContactDto).
```

Comment:
"you must set this three data to your mail" => "you must pass those to your DTO's data"